doc/source/development/contributing_environment.rst

```diff
@@ -142,14 +97,13 @@ compiler installation instructions.
 
 Let us know if you have any difficulties by opening an issue or reaching out on `Gitter <https://gitter.im/pydata/pandas/>`_.
 
-Creating a Python environment
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Option 1a: using mamba (recommended)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Now create an isolated pandas development environment:
 
-* Install either `Anaconda <https://www.anaconda.com/products/individual>`_, `miniconda
-  <https://docs.conda.io/en/latest/miniconda.html>`_, or `miniforge <https://github.com/conda-forge/miniforge>`_
-* Make sure your conda is up to date (``conda update conda``)
+* Install `mamba <https://mamba.readthedocs.io/en/latest/index.html>`_
```

Comment:
There are some references to `conda` in the section above this one. Should those also be changed or maybe linked directly to this section for clarity?

Reply:
yes, good point, thanks

the [mamba docs](https://mamba.readthedocs.io/en/latest/user_guide/mamba.html) have `conda activate`, but `mamba activate` also works fine, I've replaced `conda` with `mamba` throughout to avoid confusion

Reply:
I'd personally use this link to install mamba: https://github.com/conda-forge/miniforge#mambaforge

In my opinion this is what users should use, and it's not so obvious to find. Users may end up installing Anaconda, to then install mamba... which doesn't sound ideal.

But if you have a preference for the mamba home docs, also fine with it.

Reply:
Yeah agree that that's what people should use - it's the first link to show up on https://mamba.readthedocs.io/en/latest/installation.html , so let's link that (which'll probably be the most up-to-date reference)?

Reply:
Yeah agree that that's what people should use - it's the first link to show up on https://mamba.readthedocs.io/en/latest/installation.html , so let's link that (which'll probably be the most up-to-date reference)?